backend/open_webui/routers/auths.py

```
@@ -43,6 +41,7 @@
     AUTH0_CLIENT_SECRET,
     AUTH0_DOMAIN,
     ENABLE_LDAP,
+    DEFAULT_USER_PERMISSIONS
```

Ruff (0.8.2)

`open_webui.config.DEFAULT_USER_PERMISSIONS` imported but unused

Remove unused import

(F401)